spacialgaze-plugins/misc.js

```diff
@@ -398,4 +407,23 @@ exports.commands = {
 		this.parse("/tour create " + target + ", roundrobin");
 	},
 	rtourhelp: ["/rtour [format] - Creates a round robin tournament in the format provided."],
+
+	disableintroscroll: function (target, room, user) {
+		if (!this.can('roomowner')) return false;
+		if (!target) return this.errorReply("No Room Specified");
+		target = toId(target);
+		if (!Rooms(target)) return this.errorReply(`${target} is not a room`);
+		if (Db.disabledScrolls.has(target)) return this.errorReply(`${Rooms(target).title} has roomintro scroll disabled.`);
```

Comment:
There should be a check to make sure your not disabling an already disabled intro.